tests/test_main.py

```diff
@@ -400,6 +400,55 @@ def test_context_manager_skips_end_calls(tmp_dir):
     assert (tmp_dir / live.metrics_file).exists()
 
 
+@pytest.mark.vscode()
+@pytest.mark.parametrize("dvc_root", [True, False])
+def test_vscode_dvclive_step_completed_signal_file(tmp_dir, dvc_root, mocker):
+    signal_file = os.path.join(
+        tmp_dir, ".dvc", "tmp", "exps", "run", "DVCLIVE_STEP_COMPLETED"
```

Comment:
High-level question, how does this work when queued experiments are running in parallel?

Reply:
The file will get overwritten as each experiment completes a step. 

IMO this is okay because all we need is for an event to be created so that the extension can reliably run `exp show` at the correct time. We are not currently interrogating the contents of the file but we could use the PID to clean it up when it is stale.

Reply:
To elaborate, we have a single signal file but we could have multiple running experiments in parallel writing to the same file

Reply:
For the experiments table in the extension, we watch for file events in order to call for data updates. These watched files include params and metrics files (gathered from the `exp show` output). When there are experiments running in the queue on remote machines some of the necessary events are dropped which means the table can get stale. 

The single purpose of this file is to make sure that the extension calls `exp show` at the right time. If there are multiple experiments running then we want to call for a data update whenever one of the experiments completes a step.

With all of that said maybe the contents of the file are wrong. I was just concerned that if we didn't actually change the contents of the file that VS Code would drop those events as well.